src/styles/utils/variables.scss

```diff
@@ -0,0 +1,8 @@
+$blue-elements-color: #00acdc;
```

Comment:
Consider using a more descriptive variable name, such as '$primary-color' instead of '$blue-elements-color'